CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # [ Unreleased ]
 
-* Fixes bug where default socketpath would always be used when not using jailer (#84)
+* Updates jailer's socket path to point to the root drive.
```

Comment:
This does not make sense. Socket path should not point to a root drive.

Reply:
I'll reword to `Updates the jailer's socket path to point to the socket path in the jailer's workspace`